### PR TITLE
Fixed Issue #153: Unnecessary Outline in Input fields

### PR DESCRIPTION
**Pull request by Abhishek765**
kindly review it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,13 +35,13 @@
     "prop-types": "^15.7.2",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.0.tar.gz",
+    "react-native": "^0.64.0",
```
Comment (flyingcircle) [inline package.json]: This is incorrect, please don't change this.

File: package.json [hunk 2]
```diff
@@ -35,13 +35,13 @@
...
     "react-native-appearance": "~0.3.4",
     "react-native-elements": "https://github.com/react-native-elements/react-native-elements",
-    "react-native-gesture-handler": "~1.9.0",
+    "react-native-gesture-handler": "~1.8.0",
```
Comment (flyingcircle) [inline package.json]: Don't go backwards

File: package.json [hunk 3]
```diff
@@ -35,13 +35,13 @@
...
     "react-native-reanimated": "~1.13.2",
     "react-native-safe-area-context": "3.1.9",
-    "react-native-screens": "~2.16.1",
+    "react-native-screens": "~2.15.2",
```
Comment (flyingcircle) [inline package.json]: Yeah, you shouldn't have changed any line in this file.

**Review by flyingcircle — CHANGES_REQUESTED** [review]

**Comment by Abhishek765** [comment]
@flyingcircle Done can you check

**Pull request merged**